app/views/stats/_stats_nav.html.erb

```diff
@@ -0,0 +1,13 @@
+<div align="center" >
+    <nav class="navbar navbar-dark bg-primary">
+        <h4><a href="/stats" class="<%= current_page? stats_path%>" style="color:white;">All Stats</a></h4>
```

Comment:
What if we removed the `h4` here and the one you're on could be `background-color: #e3f2fd` while the others would just have blank backgrounds? I think that could clarify a bit. Then we should be good!

Reply:
cool, on it